src/emulator/modules/ScePower/ScePower.cpp

```diff
@@ -63,7 +63,7 @@ EXPORT(int, scePowerGetBatteryLifeTime) {
     int res;
     SDL_GetPowerInfo(&res, NULL);
     if (res == -1){
-        return INT_MAX;
+        return 0xFFFFFFFF;
```

Comment:
Reason? Is this what scePower does (did you reverse it)? Because it returns an int, not unsigned int.

Reply:
It's a quick fix, sorry :P

Reply:
If it's right, it should probably be `UINT_MAX`.

Reply:
I just noticed, isn't this function return an int? Why UINT_MAX?

Reply:
Because I assumed you changed this because you reversed the function and saw that it was returning this value.

Reply:
Said function should return UNKNOWN state error when this value is not known but this could potentially break everything running on Desktop PC, that's why i think it's better to stick with INT_MAX.